mindsdb/integrations/handlers/tpot_handler/tpot_handler.py

```diff
@@ -0,0 +1,51 @@
+from typing import Optional
+import dill
+import pandas as pd
+from mindsdb.integrations.libs.base import BaseMLEngine
+from typing import Dict, Optional
+from type_infer.infer import infer_types
+from tpot import TPOTClassifier, TPOTRegressor
+
+
+class TPOTHandler(BaseMLEngine):
+    name = "TPOT"
+    def create(self, target: str, df: Optional[pd.DataFrame] = None, args: Optional[Dict] = None) -> None:
+        if args is None:
+            args = {}
+        
+        
+        self.target=target
+
+        target_dtype=infer_types(df,0).to_dict()["dtypes"][self.target]
+
+
+        if target_dtype in ['binary','categorical','tags']:
+            model = TPOTClassifier(generations=args.get('generations', 10),
+                                       population_size=args.get('population_size', 100),
+                                       verbosity=2,
+                                       max_time_mins=args.get('max_time_mins', None),
+                                       n_jobs=args.get('n_jobs', -1))
+            
+
+        elif target_dtype in ['integer','float','quantity'] :
+            model = TPOTRegressor(generations=args.get('generations', 10),
+                                      population_size=args.get('population_size', 100),
+                                      verbosity=2,
+                                      max_time_mins=args.get('max_time_mins', None),
+                                      n_jobs=args.get('n_jobs', -1))
+        else:
+            print('Unexpected error Occur')
+
+        if df is not None:
```

Comment:
Shouldn't we raise an exception if `df is None` here? Or is there some value in a non-fitted TPOT model?

Reply:
@parthiv11 I still think we should raise an exception here, unless I'm missing something.

Reply:
done